File: iosgl/iosgl/bt/skill/vo/SkillKeyVo.m
//
//  SkillKeyVo.m
//  iosgl
//
//  Created by zhao on 17/4/2020.
//  Copyright © 2020 zhao. All rights reserved.
//

#import "SkillKeyVo.h"

@implementation SkillKeyVo
-(void)setData:(DataObjTempVo*)data;
{
    self.frame =data.frame ;
    self.url = data.url;
}
@end

File: android/app/src/main/java/com/pan3d/skill/vo/SkillKeyVo.java
package com.pan3d.skill.vo;

import com.pan3d.vo.DataObjTempVo;

public class SkillKeyVo {
    public float frame  = 0;
    public String url;

    public void setData(DataObjTempVo $data) {
        this.frame = $data.frame;
        this.url = $data.url;
    }
}
